docs/data/stashcache/install-cache.md

```diff
@@ -129,26 +129,22 @@ The mandatory variables to configure are:
 - `set resourcename = YOUR_RESOURCE_NAME`: the resource name registered with the OSG.
 
 
-Configuring the Authenticated Cache (Optional)
-----------------------------------------------
-
-The authenticated cache service is a separate instance of the StashCache cache service,
-and runs alongside the unauthenticated instance.
-You should make sure that the unauthenticated instance is functioning before setting up the authenticated instance.
-Before proceeding, make sure you have followed the [prerequisite steps](#before-starting)).
-
-
 ### Add a service certificate
```

Comment:
I'd strongly recommend renaming this title. We're not really adding service certificate. Service certificate is something different (typicaly consist of prefix in CN like `CN=service/FQDN`). We just do a trick - copy hostcert to another specific location that used only by xrootd UID. How about we name this section "Clone hostcert for xrootd service".

Reply:
How about "Ensure the xrootd service has a certificate"?

Reply:
Sure, totally fine with me as long as it doesn't say "service certificate" anywhere :).